pkg/applicationserver/grpc_deviceregistry_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	. "go.thethings.network/lorawan-stack/pkg/applicationserver"
 	"go.thethings.network/lorawan-stack/pkg/auth/rights"
 	"go.thethings.network/lorawan-stack/pkg/component"
+	componenttest "go.thethings.network/lorawan-stack/pkg/component/test"
```

Comment:
TBH I think we should just name the package itself `componenttest`. I don't see a use case where the package would be imported as `test`